cumulusci/cli/cci.py

```diff
@@ -843,32 +867,63 @@ def service_connect():
 
 
 @service.command(name="info", help="Show the details of a connected service")
+@click.argument("service_type")
 @click.argument("service_name")
 @click.option("--plain", is_flag=True, help="Print the table using plain ascii.")
 @pass_runtime(require_project=False, require_keychain=True)
-def service_info(runtime, service_name, plain):
+def service_info(runtime, service_name, service_type, plain):
     try:
         plain = plain or runtime.universal_config.cli__plain_output
-        service_config = runtime.keychain.get_service(service_name)
+        service_config = runtime.keychain.get_service(service_type, service_name)
         service_data = [["Key", "Value"]]
         service_data.extend(
             [
                 [click.style(k, bold=True), str(v)]
                 for k, v in service_config.config.items()
+                if k != "service_name"
             ]
         )
         wrap_cols = ["Value"] if not plain else None
-        service_table = CliTable(service_data, title=service_name, wrap_cols=wrap_cols)
+        service_table = CliTable(
+            service_data, title=f"{service_type}/{service_name}", wrap_cols=wrap_cols
+        )
         service_table._table.inner_heading_row_border = False
         service_table.echo(plain)
     except ServiceNotConfigured:
         click.echo(
-            "{0} is not configured for this project.  Use service connect {0} to configure.".format(
-                service_name
-            )
+            f"{service_name} is not configured for this project.  Use service connect {service_name} to configure."
         )
 
 
+@service.command(name="default", help="Show the details of a connected service")
+@click.argument("service_type")
+@click.argument("service_name")
+@pass_runtime(require_project=False, require_keychain=True)
+def service_default(runtime, service_type, service_name):
+    try:
+        runtime.keychain.set_default_service(service_type, service_name)
+    except ServiceNotConfigured as e:
+        click.echo(f"An error occurred setting the default service: {e}")
+        return
+
+    click.echo(f"'{service_name}' set as the default for {service_type} services.")
```

Comment:
Doesn't this need a way to set a project-level override of the default?

Reply:
Confirmed that works, but can we show a message that indicates which default was set (e.g. "for all projects" vs "for this project").